Project.toml

```diff
@@ -38,14 +35,11 @@ DocStringExtensions = "0.8, 0.9"
 Insolation = "0.6"
 JLD2 = "0.4"
 NCDatasets = "0.11, 0.12"
-OrdinaryDiffEq = "5, 6"
 Plots = "1.39.0"
```

Comment:
Is this patch version still necessary?

Reply:
I'm not sure. Do you know why it was added in #482 ?

Reply:
The new version was breaking the HDF5 dep. We can keep it for now, given that Atmos is having related issues and we are stable.